roles/wordpress-setup/templates/wordpress-site.conf.j2

```diff
@@ -7,7 +7,7 @@ server {
   listen 80;
   {% endif %}
 
-  server_name  {{ item.value.site_hosts | join(' ') }};
+  server_name  {% for host in item.value.site_hosts %} {{ host }} {% if item.value.multisite.subdomains %} *.{{ host }} {% endif %} {% endfor %};
```

Comment:
@louim Yes, that IP routing behavior is what I have been seeing/using.  

@fullyint Landrush does not allow executing this with a wildcard host:

`config.landrush.host host, PRIVATE_IP`

Why? No idea. I didn't find anything in a cursory look through the Landrush docs or issues.

The fork where it doesn't fail that you linked configures Landrush/the Vagrant box in a different manner versus the method presented in the wiki. Among several things, it doesn't use `site_hosts` to generate the subdomains for Landrush/the Vagrant box - and therefore doesn't encounter the DNS armageddon that results. Instead it uses configuration code that seems more specifically adapted to its environments/needs.

When I was testing things, it seemed to me that the most basic choice was between removing the wildcard from `site_hosts` and leaving the wiki as is on one hand or using the wildcard and significantly reworking the Landrush configuration code on the other. As my Landrush experience is minimal, I went with the former - but perhaps another approach is better/more elegant?